activation/poet.go

```diff
@@ -108,19 +126,52 @@ func (c *HTTPPoetClient) Submit(ctx context.Context, challenge []byte, signature
 	if err := c.req(ctx, "POST", "/submit", &request, &resBody); err != nil {
 		return nil, err
 	}
-
-	return &types.PoetRound{ID: resBody.RoundId, ChallengeHash: resBody.Hash}, nil
+	roundEnd := time.Time{}
+	if resBody.RoundEnd != nil {
+		roundEnd = time.Now().Add(resBody.RoundEnd.AsDuration())
```

Comment:
curious about the decision to communicate round end as duration and not absolute time?

Reply:
I figured that it will be simpler to pass duration as it is an absolute value (no clock drift, no timezones etc.). The downside is that the calculated round end will be off (late) by an unknown value due to the time it takes for the GRPC response to arrive. We don't need an exact round end time. An estimation of when the proof _should_ be available is enough.

Reply:
i pointed this out because using duration, the rpc response is not informational at a later time, i.e. no good for debugging, or if the data is stored somewhere and then relayed. i think it better to use absolute time (utc) in req/rep in general.
either way we will have clock drift.

Reply:
for later consideration. no need to change now.